src/index.html

```diff
@@ -7,8 +7,41 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img src="images/logo.png" alt="moyo logo">
+      </a>
+      <nav class="navigation">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="index.html" class="is-active">apple</a>
+          </li>
+          <li class="nav_item">
```

Comment:
Add a blank line between the links
```suggestion
          </li>
          
          <li class="nav_item">
```

Reply:
done